build.gradle

```diff
@@ -90,10 +91,12 @@ sourceSets {
 sourceSets.main.compileClasspath += sourceSets.java9.output.classesDirs;
 
 dependencies {
-	implementation 'com.offbytwo:docopt:0.6.0.20150202'
+	implementation 'dev.jbang:devkitman:0.1.2'
+
+//	implementation 'com.offbytwo:docopt:0.6.0.20150202'
```

Comment:
This probably shouldn't be here, but it seemed unused to I commented it out and it slipped into the PR. If you want I can take it out.

Reply:
lol - thats a ancient leftover from very early jbang...never needed. lets just nuke it.